sql/updates/world/3.3.5/2016_05_15_15_world.sql

```diff
@@ -0,0 +1,54 @@
+(@BunthenGossip,4918);
+
+-- Conditions for @Bunthen + @Silva gossip options
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` IN (14,15) AND `SourceGroup` IN (@Bunthen, @Silva);
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`,`SourceGroup`,`SourceEntry`,`SourceId`,`ElseGroup`,`ConditionTypeOrReference`,`ConditionTarget`,`ConditionValue1`,`ConditionValue2`,`ConditionValue3`,`NegativeCondition`,`ErrorType`,`ErrorTextId`,`ScriptName`,`Comment`) VALUES
+(14,@SilvaGossip,4913,0,0,15,0,1024,0,0,1,0,0,'','Show npc_text.id 4913 if Player is NOT a Druid'),
+(14,@SilvaGossip,4914,0,0,15,0,1024,0,0,0,0,0,'','Show npc_text.id 4914 if Player is a Druid'),
+(14,@SilvaGossip,4915,0,0, 6,0,  67,0,0,0,0,0,'','Show npc_text.id 4915 if Player belongs to the Horde team'),
+(15,@SilvaGossip,   0,0,0,15,0,1024,0,0,0,0,0,'','Show gossip menu option 4041 id 0 if Player is a Druid'),
+(15,@SilvaGossip,   1,0,0, 9,0,  30,0,0,0,0,0,'','Show gossip menu option 4041 id 1 if Quest 30 is taken (active)'),
+(15,@SilvaGossip,   2,0,0, 9,0, 272,0,0,0,0,0,'','Show gossip menu option 4041 id 2 if Quest 272 is taken (active)'),
+(14,@BunthenGossip,4916,0,1,15,0,1024,0,0,1,0,0,'','Show npc_text.id 4916 if Player is NOT a Druid'),
+(14,@BunthenGossip,4918,0,1,15,0,1024,0,0,0,0,0,'','Show npc_text.id 4918 if Player is a Druid'),
+(14,@BunthenGossip,4917,0,1, 6,0, 469,0,0,0,0,0,'','Show npc_text.id 4917 if Player belongs to the Alliance team'),
+(15,@BunthenGossip,   0,0,1,15,0,1024,0,0,0,0,0,'','Show gossip menu option 4042 id 0 if Player is a Druid'),
+(15,@BunthenGossip,   1,0,1, 9,0,  30,0,0,0,0,0,'','Show gossip menu option 4042 id 1 if Quest 30 is taken (active)'),
+(15,@BunthenGossip,   2,0,1, 9,0, 272,0,0,0,0,0,'','Show gossip menu option 4042 id 2 if Quest 272 is taken (active)');
+
+-- SmartAI script for @Bunthen & @Silva
+DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryorguid` IN (@Bunthen, @Silva);
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
+(@Bunthen,0, 0,1, 62,0,100,0,@BunthenGossip,0,0,0, 72,  0,   0,0,0,0,0,7,0,0,0,0,0,0,0,"Bunthen Plainswind - on Gossip Option 0 selected - Close Gossip"),
+(@Bunthen,0, 1,0, 61,0,100,0,             0,0,0,0, 52,316,   0,0,0,0,0,7,0,0,0,0,0,0,0,"Bunthen Plainswind - on Gossip Option 0 selected - Activate Taxi Path 316"),
+(@Bunthen,0, 2,3, 62,0,100,0,@BunthenGossip,1,0,0, 72,  0,   0,0,0,0,0,7,0,0,0,0,0,0,0,"Bunthen Plainswind - on Gossip Option 1 selected - Close Gossip"),
+(@Bunthen,0, 3,2, 61,0,100,0,             0,0,0,0, 98,  0,5373,0,0,0,0,7,0,0,0,0,0,0,0,"Bunthen Plainswind - on Gossip Option 1 selected - Send Gossip 5373"),
```

Comment:
Right, thanks. I suppose I got confused by the conditions and SAI handling. Will fix.